### PR TITLE
[dependencies]

### LINKED ISSUE #26: [dependencies] Update django-leaflet
Upgrade django-leaflet to the latest available version, ensuring that the outdoor and indoor map features still work as expected.

**Reserved for GCI.**

### PR DESCRIPTION
**Pull request by IAmAMissingColon**
Added support of new version of django-leaflet

Fixes #26

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ python:
 branches:
   only:
     - master
+    - requirements
```
Comment (nemesisdesign) [inline .travis.yml]: you don't need to do this, travis is configured to automatically create builds for pull requests, please remove this line.

Comment (IAmAMissingColon) [reply .travis.yml]: Fixed it :)

**Comment by IAmAMissingColon** [comment]
> Good start @IAmAMissingColon!
> 
> See my comment below.
> 
> Have you tested the running application?
> 
> Can you send a few screenshots of the geographic and floorplan map sections of a device which has an indoor location configured please?

Sorry for late reply :sweat: was in college 

So. I checked the leaflet version and it shows 0.25.0

but when I run the server and go to geographic and floor plan section it shows nothing :thinking: 

Am I doing something wrong !? 

please tell me so I can improve :smile:

**Comment by IAmAMissingColon** [comment]
Screenshot_20191203_165417

**Comment by IAmAMissingColon** [comment]
Screenshot_20191203_163233

**Comment by IAmAMissingColon** [comment]
@nemesisdesign , @atb00ker  Something like that ? Please correct if wrong :smile:

**Comment by IAmAMissingColon** [comment]
> Double checked and it works, thanks!

Thank You :smile:

**Pull request merged**